frontend/ui/opdsparser.lua

```diff
@@ -78,16 +78,11 @@ function OPDSParser:parse(text)
     text = text:gsub('<content type="xhtml">.-</content>', '')
     -- luxl doesn't handle XML comments, so strip them
     text = text:gsub("<!%-%-.-%-%->", "")
-    -- luxl prefers <br />, the other two forms are valid in HTML, but will kick luxl's ass
+    -- luxl is also particular about the syntax for self-closing, empty & orphaned tags...
+    text = text:gsub("<(%a+)/>", "<%1 />")
```

Comment:
Characters like `=`, `'` and `"` count as letters? This can be things like `<img src="some really weird but presumably escaped characters" bla-weirdness="blabla"/>`.

Reply:
I sure hope not, I'm blindly assuming it's basically `[a-Z]` ^^.

Reply:
Well, then there are a lot of self-closing tags that'll still close issues, surely? ;-) But it's more efficient than manually adding the next `<hr/>`, yes.

Reply:
```
> foo="<blah/>" 
> print(foo:gsub("<(%l+)/>", "<%1 />"))
<blah />        1
> foo='<img src="wheee"/>'
> print(foo:gsub("<(%l+)/>", "<%1 />"))
<img src="wheee"/>      0
```

IIRC, as long as there are attributes, luxl is fine with the space-less `/>`, so this ought to do ;).

Reply:
> as long as there are attributes, luxl is fine with the space-less 

Well, all right then. O_o